portafly/src/components/AppLayout.tsx

```diff
@@ -20,11 +20,7 @@ export const AppLayoutContext = React.createContext<IAppLayoutContext>({
   setBreadcrumb: () => 0
 })
 
-export interface IAppLayoutProps
-  extends Pick<PageHeaderProps, 'logo'>,
-  Pick<PageHeaderProps, 'logoProps'>,
-  Pick<PageHeaderProps, 'avatar'>,
```

Comment:
What happens with the avatar? screenshot?

Reply:
The Avatar goes now in `headerTools`. But we don't have an Avatar so nothing changes for us.